bia_export/cli.py

```diff
@@ -136,114 +56,36 @@ def bia_image_to_export_image(image, study, use_cache=True):
     if use_cache and output_fpath.exists():
         return ExportImage.parse_file(output_fpath)
 
-    reps_by_type = {rep.type: rep for rep in image.representations}
-
-    ome_zarr_uri = reps_by_type["ome_ngff"].uri[0]
-    im = ome_zarr_image_from_ome_zarr_uri(ome_zarr_uri)
-
-    try:
-        thumbnail_uri = reps_by_type["thumbnail"].uri[0]
-    except KeyError:
-        thumbnail_uri = ""
-
-    # FIXME - should generalise this
-    source_image_uuid = image.attributes.get("source_image_uuid", None)
-    source_image_thumbnail_uri = image.attributes.get(
-        "source_image_thumbnail_uri", None
-    )
-    overlay_image_uri = image.attributes.get("overlay_image_uri", None)
-
-    export_im = ExportImage(
-        uuid=image.uuid,
-        name=Path(image.name).name,
-        alias="IM1",
-        original_relpath=image.original_relpath,
-        thumbnail_uri=thumbnail_uri,
-        study_accession_id=study.accession_id,
-        study_title=study.title,
-        release_date=study.release_date,
-        itk_uri=ITK_BASE + reps_by_type["ome_ngff"].uri[0],
-        vizarr_uri=VIZARR_BASE + reps_by_type["ome_ngff"].uri[0],
-        sizeX=im.sizeX,
-        sizeY=im.sizeY,
-        sizeZ=im.sizeZ,
-        sizeT=im.sizeT,
-        sizeC=im.sizeC,
-        PhysicalSizeX=im.PhysicalSizeX,
-        PhysicalSizeY=im.PhysicalSizeY,
-        PhysicalSizeZ=im.PhysicalSizeZ,
-        source_image_uuid=source_image_uuid,
-        source_image_thumbnail_uri=source_image_thumbnail_uri,
-        overlay_image_uri=overlay_image_uri,
-        attributes=filter_image_attributes(image),
-    )
+    image_acquisitions = []
+    specimens = []
+    biosamples = []
 
-    # TODO: Refactor so titles and child uuids obtained in same API call - use for loop
-    # Get BioSample, Specimen and ImageAcquisition
-    specimen_uuids = []
-    specimen_uuids.extend(
+    # image->image_acquisition is 1->many, though usually only 1 acquisition per image
+    image_acquisitions.extend(
         [
-            rw_client.get_image_acquisition(image_acquisition_uuid).specimen_uuid
+            rw_client.get_image_acquisition(image_acquisition_uuid)
```

Comment:
ahh - and I (@kbab) am the one that is using rw_client directly!!! Happy to discuss.

Reply:
Yeah, i was somewhat hoping i could move these kinds of methods out to bia_client_utils, here the method was almost so simple that it would warrant a more complex abstraction?